stage1/usr_from_kvm/files.mk

```diff
@@ -0,0 +1,33 @@
+$(call setup-stamp-file,UFKF_STAMP)
+UFKF_DIR := $(MK_SRCDIR)/files
+UFKF_F1_IN_ACI := $(ACIROOTFSDIR)/etc/passwd
+UFKF_SRC_F1 := $(UFKF_DIR)/passwd
+UFKF_F2_IN_ACI := $(ACIROOTFSDIR)/etc/shadow
+UFKF_SRC_F2 := $(UFKF_DIR)/shadow
+UFKF_F3_IN_ACI := $(ACIROOTFSDIR)/usr/lib64/systemd/system/sshd.service
+UFKF_SRC_F3 := $(UFKF_DIR)/sshd.service
+UFKF_F4_IN_ACI := $(ACIROOTFSDIR)/etc/group
+UFKF_SRC_F4 := $(UFKF_DIR)/group
+UFKF_F5_IN_ACI := $(ACIROOTFSDIR)/etc/ssh/sshd_config
+UFKF_SRC_F5 := $(UFKF_DIR)/sshd_config
+
+STAGE1_STAMPS += $(UFKF_STAMP)
+
+STAGE1_INSTALL_FILES += \
+	$(UFKF_SRC_F1):$(UFKF_F1_IN_ACI):- \
+	$(UFKF_SRC_F2):$(UFKF_F2_IN_ACI):- \
+	$(UFKF_SRC_F3):$(UFKF_F3_IN_ACI):- \
+	$(UFKF_SRC_F4):$(UFKF_F4_IN_ACI):- \
+	$(UFKF_SRC_F5):$(UFKF_F5_IN_ACI):-
+
+STAGE1_INSTALL_DIRS += \
+	$(ACIROOTFSDIR)/etc:0755 \
+	$(ACIROOTFSDIR)/root/.ssh:0700 \
```

Comment:
Nothing depends on that directory, so it will not be created. Anyway, you create it in the code.